src/components/beginComponent.js

```diff
@@ -34,6 +34,13 @@ module.exports = function beginComponent(componentsContext, component, isSplitCo
         return componentDef;
     }
 
+    if (isImplicitComponent === true) {
```

Comment:
Thoughts on moving this conditional above the one that is currently above it? Micro-optimization, but there's probably no sense in doing the bitwise operations above if this is `true`.

Reply:
It's actually going to be more common for the if condition on line 32 to evaluate to `true` so it's better to keep the `if (isImplicitComponent === true)` lower. Whether or not a UI component is implicit only matters if it is a top-level UI component outside of UI component that will re-render in the browser.